Three package-info.class files with bytecode version 53 (JDK 9) is added to jar

In one of our projects use the extra maven enforcer rule `enforceBytecodeVersion` to ensure that artifacts we believe to be JDK 8 compliant actually are.

This fails when running against threeten-extra 1.7.1:

```
[INFO] --- maven-enforcer-plugin:3.1.0:enforce (enforce) @ bom-integration-test ---
[INFO] Adding ignore: module-info
[INFO] Restricted to JDK 8 yet org.threeten:threeten-extra:jar:1.7.1:compile contains org/threeten/extra/chrono/package-info.class targeted to JDK 9
[ERROR] Rule 3: org.apache.maven.plugins.enforcer.EnforceBytecodeVersion failed with message:
Found Banned Dependency: org.threeten:threeten-extra:jar:1.7.1
```

We can get around this with exclusions
I noticed that the package-info.class files are not in the 1.7.0 jar file, but they are included in 1.7.1 with bytecode version 53. 
Is this intentional?

--- a/pom.xml
+++ b/pom.xml
@@ -809,6 +809,7 @@
             <!-- setup defaults for compile and testCompile -->
             <configuration>
               <release>8</release>
+              <createMissingPackageInfoClass>false</createMissingPackageInfoClass>
             </configuration>
           </plugin>
         </plugins>

